src/common/helpers/utils.ts

```diff
@@ -53,38 +55,37 @@ export function convertIpfsLink(link: string): string {
   let metadata = link;
 
   if (metadata.startsWith("ipfs://ipfs/")) {
-    metadata = "https://flk-ipfs.xyz/ipfs" + metadata.slice(11);
+    metadata = "https://ipfs1.fiit.stuba.sk/ipfs" + metadata.slice(11);
   }
   else if (metadata.startsWith("https://ipfs.io/ipfs/")) {
-    metadata = "https://flk-ipfs.xyz/ipfs/" + metadata.slice(16);
+    metadata = "https://ipfs1.fiit.stuba.sk/ipfs" + metadata.slice(16);
   }
   else if (metadata.startsWith("ipfs:/")) {
-    metadata = "https://flk-ipfs.xyz/ipfs/" + metadata.slice(7);
+    metadata = "https://ipfs1.fiit.stuba.sk/ipfs" + metadata.slice(7);
   }
-  else if (!metadata.startsWith("https://flk-ipfs.xyz/")) {
-    metadata = "https://flk-ipfs.xyz/ipfs/" + metadata;
+  else if (!metadata.startsWith("https://ipfs1.fiit.stuba.sk")) {
+    metadata = "https://ipfs1.fiit.stuba.sk/ipfs" + metadata;
   }
   return metadata;
 }
 
 export async function fetchMetadataFromIPFS(metadatalink: string): Promise<string> {
   // Fetch metadata from IPFS
   // Return the metadata 
+  const configService = new ConfigService<AppConfig>();

```

Comment:
In order to use ConfigService you need dependency injection which is only available in Module, Service, Controller...
You can however move this to the newly created AppConfigService in ConfigModule.